views/js/GoogleAnalyticActionLib.js

```diff
@@ -156,7 +157,7 @@ var GoogleAnalyticEnhancedECommerce = {
 
 		//this.add(Product);
 		ga('ec:setAction', 'purchase', Order);
-		ga('send', 'pageview', {
+		ga('send', 'event','Transaction','purchase', {
```

Comment:
According to the documentation: Enhanced Ecommerce parameters must be sent with an existing hit (e.g. pageview, event). So it should be ok to send only event.